packages/modal/stories/Modal.stories.tsx

```diff
@@ -66,343 +67,458 @@ export default {
   }
 } as Meta;
 
+const Template1 = args => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  function handleChange() {
+    setIsOpen(!isOpen);
+  }
+
+  return (
+    <div>
+      <div style={{ textAlign: "center" }}>
+        <PrimaryButton onClick={handleChange}>Open Modal</PrimaryButton>
+      </div>
+
+      <DialogModal
+        isOpen={isOpen}
+        onClose={handleChange}
+        title="I am modal"
+        {...args}
+      >
+        <ModalContent />
+      </DialogModal>
+    </div>
+  );
+};
+
 export const _DialogModal = {
-  render: args => (
-    <ModalStoryContainer>
-      {({ isOpen, onClose }) => (
-        <DialogModal
-          isOpen={isOpen}
-          onClose={onClose}
-          title="I am modal"
-          {...args}
-        >
-          <ModalContent />
-        </DialogModal>
-      )}
-    </ModalStoryContainer>
-  )
+  render: Template1
 };
 
 const confirmIcon = {
   shape: SystemIcons.CircleInformation,
   color: blue
 };
 
+const Template2 = args => {
```

Comment:
Instead of all of these Templates, it would be best to try to work around the new way to write stories that use hooks. 
You can read up on it further here in the [Storybook Docs](https://storybook.js.org/docs/react/writing-stories/introduction#working-with-react-hooks).

Let me know if you have any questions!

Reply:
Sure thing!

Reply:
Can you be more specific about this request ? Should I remove the templates altogether and directly export the React elements? @nataliepina

Reply:
I think what you've done now is fine, we will probably migrate to this new documentation style in the future but this aligns with other stories.